ferveo/src/dkg/pv.rs

```diff
@@ -1,717 +0,0 @@
-use std::collections::BTreeMap;
```

Comment:
I don't follow the development in this repo, but I'm just curious, how does this removal fit with the topic of the PR?

Reply:
It doesn't fit here, it's just something I noticed. I admit it should be a separate change to avoid any confusion with this PR.

Would you be interested in following the development here or reviewing PRs in the future?

Reply:
I would, but I cannot guarantee that I will be available.